### PR TITLE
Fix restore of k8s UDP services

### PR DESCRIPTION
**Pull request by tgraf**
This change is [Reviewable](https://reviewable.io/reviews/cilium/cilium/6970)

### CODE REVIEW

**Comment by tgraf** [comment]
test-me-please

Runtime test failed before the existing service restore test broke. It worked by accident as only TCP was tested. Added a fix and update the commit message.

**Review by nebril — APPROVED** [review]

**Comment by tgraf** [comment]
test-me-please

**Review by jrfastab — APPROVED** [review]

**Comment by tgraf** [comment]
k8s-1.10.K8sServicesTest Checks ClusterIP Connectivity Checks service on same node is failing consistently. Investigating.

**Comment by tgraf** [comment]
test-me-please

**Review by ianvernon — APPROVED** [review]
LGTM, easy to backport to v1.4 with minimal impact.

**Comment by ianvernon** [comment]
@tgraf IMO we should backport this to v1.3 as well since it is LTS

**Comment by tgraf** [comment]
> @tgraf IMO we should backport this to v1.3 as well since it is LTS

Agreed

File: pkg/maps/lbmap/lbmap.go [hunk 1]
```diff
@@ -530,7 +530,7 @@ func serviceKeynValue2FEnBE(svcKey ServiceKey, svcValue ServiceValue) (*loadbala
 	}
 
 	feL3n4Addr := serviceKey2L3n4Addr(svcKey)
-	beLBBackEnd := loadbalancer.NewLBBackEnd(loadbalancer.TCP, beIP, bePort, beWeight)
+	beLBBackEnd := loadbalancer.NewLBBackEnd(loadbalancer.NONE, beIP, bePort, beWeight)
```
Comment (aanm) [inline pkg/maps/lbmap/lbmap.go:533]: Not blocker but in reality this should be `loadbalancer.ANY`

Comment (tgraf) [reply pkg/maps/lbmap/lbmap.go:533]: Agreed but I'm not going to refactor code in this fix ;-)

**Comment by tgraf** [comment]
test-me-please

**Comment by tgraf** [comment]
Hit #6947 in the PR-GInkgo-Tests-K8s

**Pull request merged**